### PR TITLE
fix job to build image

### PR DESCRIPTION
**Pull request by Bharath92**
https://github.com/Shippable/core.prep/issues/1

### CODE REVIEW

File: shippable.yml [hunk 1]
```diff
@@ -106,7 +106,7 @@ jobs:
       - TASK:
           script:
             - pushd $(shipctl get_resource_state "core_prep_src")
-            - ./genBuildImage.sh
+            - ./genBuildImage.sh core_prep
```
Comment (shrivara) [inline shippable.yml]: I think I think the script we are referring to the resource as `core_prep_repo` but here its actually `core_prep_src`.

Comment (shrivara) [reply shippable.yml]: Also, I think the image is supposed to be the context.

**Comment by rageshkrishna** [comment]
I don't think this is a valid change. Context is supposed to come from the job_triggered_by_name env var. Why do you need to initialize it to `core_prep`, and why does that need to come as a script parameter? This doesn't make a lot of sense to me.

File: shippable.yml (hunk 1 above)
Comment (shrivara) [reply shippable.yml]: Looking at the full script, we just need to run the job with the correct env set.

**Pull request closed without merging**